python/docker-compose.yml

```diff
@@ -1,18 +1,10 @@
-version: '2.1'
+version: "3"
 services:
-  flask:
-    image: quay.io/azavea/flask:1.0-python3.6-slim
-    working_dir: /usr/src
+  database:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
It felt valuable to use the `Dockerfile` in both examples, despite the added moving parts vs. using the Flask container image.